agent/php_error.c

```diff
@@ -387,12 +387,17 @@ void nr_php_error_install_exception_handler(TSRMLS_D) {
 static zval* nr_php_error_exception_stack_trace(zval* exception TSRMLS_DC) {
   zval* trace;
 
-  trace = nr_php_call(exception, "getTrace");
-  if (!nr_php_is_zval_valid_array(trace)) {
-    nr_php_zval_free(&trace);
+  trace = nr_php_get_zval_base_exception_property(exception, "trace");
+  Z_ADDREF_P(trace);
```

Comment:
Adding a ref count here was desired because the zval is directly returned. In other cases where the zval is not returned, the ref count is left alone for performance

Reply:
Would it make more sense to check if the returned `trace` value is `NULL` before incrementing the refcount? How does `Z_ADDREF_P` and `Z_DELREF_P` behave when fed a `NULL` value?